tests/integration/targets/lookup_etcd3/aliases

```diff
@@ -0,0 +1,5 @@
+shippable/posix/group4
+destructive
+needs/file/tests/utils/constraints.txt
+skip/aix
+skip/python2.6  # lookups are controller only, and we no longer support Python 2.6 on the controller
```

Comment:
It's probably also a good idea to add `skip/freebsd`.

Reply:
yes it is. I did the fix, but not pushed yet as I'm thinking of smarter approach for integration test